packages/core/src/App/Containers/AccountSwitcher/account-switcher-account-list.jsx

```diff
@@ -76,7 +76,7 @@ const AccountList = ({
                             {loginid}
                         </div>
                     </span>
-                    {has_reset_balance ? (
+                    {has_reset_balance && balance < 10000 ? (
```

Comment:
@hamid-deriv Can’t we get this value dynamically from somewhere? or we don’t have that data? 🤔

Reply:
@farzin-fs  I searched for that but I got nothing, and because there was mentioned in the task exactly this balance, I thought this is a fixed number. I will ask it to be sure about this.

Reply:
Add your new condition as part of `has_reset_balance`.

Reply:
@farzin-fs I asked it and confirmed that this amount and currency are static.

Reply:
@matin-binary sorted.